qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -422,6 +423,34 @@ class BaseSpecification extends Specification {
         orchestrator.deleteSecret("gcr-image-pull-secret", Constants.ORCHESTRATOR_NAMESPACE)
     }
 
+    static addRedHatImagePullSecret(ns = Constants.ORCHESTRATOR_NAMESPACE) {
+        if (!Env.IN_CI && (Env.get("REDHAT_USERNAME", null) == null ||
+                           Env.get("REDHAT_PASSWORD", null) == null)) {
```

Comment:
nit: `null` is the default value so don't need to pass it
```suggestion
        if (!Env.IN_CI && (Env.get("REDHAT_USERNAME") == null ||
                           Env.get("REDHAT_PASSWORD") == null)) {
```

Reply:
Thanks! I just mimicked addGCRImagePullSecret above, without checking the detailed behavior of Env.get(). Suggestion applied